core/messagehelper.go

```diff
@@ -2,20 +2,19 @@ package core
 
 import (
 	"github.com/trivago/tgo/tlog"
-	"strings"
 )
 
+const APPLY_TO_PAYLOAD = "payload"
```

Comment:
Constants should follow the golang formatting rules:
http://stackoverflow.com/questions/22688906/go-naming-conventions-for-const